contrib/zsh/_polybar

```diff
@@ -21,7 +22,8 @@ _polybar() {
     "($C)"{-c,--config=}'[Path to the configuration file]:configuration file:_files' \
     "($R)"{-r,--reload}'[Reload when the configuration has been modified]' \
     "($D $R $M $W $S)"{-d,--dump=}'[Print parameter value in bar section and exit]:parameter name' \
-    "($M $D $R $W $S)"{-m,--list-monitors}'[Print list of available monitors and exit]' \
+    "($MM $M $D $R $W $S)"{-m,--list-monitors}'[Print list of available monitors and exit (Removes cloned monitors)]' \
```

Comment:
Might be better to say 'excluding' rather than 'removes'. Also should end with 'and exit' to keep everything consistent.